### PR TITLE
feat: info command

### PR DESCRIPTION
**Pull request by ChungZH**
image

写的代码很菜...

### CODE REVIEW

**Comment by ChungZH** [comment]
image
显示多个

File: tools/baulk/commands.info.cc [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    return 1;
+  }
+  bela::error_code ec;
+  auto locker = baulk::BaulkCloser::BaulkMakeLocker(ec);
+  if (!locker) {
+    bela::FPrintF(stderr, L"baulk info: \x1b[31m%s\x1b[0m\n", ec.message);
+    return 1;
+  }
+  if (!baulk::BaulkInitializeExecutor(ec)) {
+    baulk::DbgPrint(L"unable initialize compiler executor: %s", ec.message);
```
Comment (fcharlie) [inline tools/baulk/commands.info.cc:72]: baulk::BaulkInitializeExecutor is only used to initialize the msvc environment (to generate the launcher), there is no need to call.

这里不需要调用 baulk::BaulkInitializeExecutor

**Comment by ChungZH** [comment]
@fcharlie 老鸽你快来看看 😂

File: tools/baulk/commands.info.cc [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  auto pkg = baulk::bucket::PackageMetaEx(name, ec);
+  if (!pkg) {
+    bela::FPrintF(stderr, L"\x1b[31mbaulk: %s\x1b[0m\n", ec.message);
+    return 1;
+  }
+  auto lopkg = baulk::bucket::PackageLocalMeta(name, oec);
+  bela::FPrintF(stderr, L"Name: \x1b[32m%s\x1b[0m\n"
+                         "Bucket: \x1b[34m%s\x1b[0m\n"
+                         "Description: %s\n"
+    , pkg->name, pkg->bucket, pkg->description);
```
Comment (fcharlie) [inline tools/baulk/commands.info.cc:20]: 这里可以考虑按照 `Description` 的宽度进行空格填充，保持输出的对齐。

Comment (ChungZH) [reply tools/baulk/commands.info.cc:20]: image

**Pull request merged**